### ISSUE TITLE
Deprecation warning when not specifying headers style.

### ISSUE THREAD

**Issue by lukesahula** [DeepSpace2/StyleFrame#24]
Hi! First of all I want to thank you for the great work you put in this, so thanks!

I guess this is just a small problem, but when I don't specify styles for headers, I get the following warning upon calling to_excel method.

_"Directly passing style specifiers\n('bg_color', 'bold', etc.) is deprecated.\nPass a StyleFrame.Styler object instead as styler_obj."_

Python 3.5.2
StyleFrame 0.2
pandas 0.19.2
openpyxl 2.2.5

**Comment by DeepSpace2** [comment]
Thanks for pointing that out.
I fixed it by initializing all the style specifiers to `None` in `apply_headers_style`.

This way we can check if any of them is not `None` (then displaying the deprecation warning). Only then we set the default style.

```python
def apply_headers_style(self, styler_obj=None, bg_color=None, bold=None, font=None, font_size=None,
                        font_color=None, protection=None, number_format=None,
                        underline=None):

.
.
.
     elif any((bg_color, bold, font, font_size, font_color, protection, number_format, underline)):
         warnings.warn(DEPRECATION_MSG, DeprecationWarning)

     # applying "default" header style. TODO: Delete when removing support for directly passing
     # style specifiers
     # see issue #24: https://github.com/DeepSpace2/StyleFrame/issues/24
     bg_color = bg_color or utils.colors.white
     bold = bold or True
     font = font or "Arial"
     font_size = font_size or 12
     font_color = font_color or utils.colors.black
     protection = protection or False
     number_format = number_format or utils.number_formats.general
     underline = underline or None
```

I've just released version 0.2.1 that includes this change.
To install it run `$ pip install StyleFrame -U`.

**Note:** The deprecation warning is still showing when running the tests, which is an expected behavior.

**Comment by lukesahula** [comment]
Wow, thanks for the quick fix!
It seems to be working, so I'm closing the issue.
